### ISSUE TITLE
Frazier recommendations

### ISSUE THREAD

**Issue by fraziermork** [Mumbleskates/portfolio#10]
The code all seems fairly clean at this point, so these recommendations are mostly from a design perspective. 

The font is sort of ugly--maybe use something other than just arial. 
I would use something other than black, orange, and grey. If you do decide to keep this color scheme, at minimum you should make the grey lighter. 
I'm not really sure how you aligned your text for your title and your navbar, but it's current placement violates most of the principles of design because it isn't lined up with anything else. Either left-align it or center it, but don't have it centered weirdly on the left side. I think it's because all of this is in a list and the list has a left margin. 

You don't really have anything in your projects list except for placeholders, and you don't really have anything in your about section at all besides a few links. Your social link goes to facebook.com too, not to your profile.

**Comment by Mumbleskates** [comment]
Yes, good pointers. Thanks for enumerating these; I've been sort of thinking on what to do with the nav bar, it's basically a very basic placeholder at this point and definitely doesn't adhere to any sort of design; ideally this will all get iterated on a few times.
